### PR TITLE
Jf 22 m1 implement http in spring

### PR DESCRIPTION
**Pull request by jamesfernando94**
added HTTP methods for controllers

### CODE REVIEW

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class LocationController {
+
+	private LocationServiceImpl locationService;
+
+	@Autowired
+	public LocationController(LocationServiceImpl locationService) {
+		this.locationService = locationService;
+	}
+
+	@RequestMapping(value = {"","/"},method = RequestMethod.POST)
```
Comment (Gregam3) [inline backend/src/main/java/asegroup1/api/controllers/LocationController.java]: Change to `@PostMapping`

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	public @ResponseBody
```
Comment (Gregam3) [inline backend/src/main/java/asegroup1/api/controllers/LocationController.java]: `@ResponseBody not necessary`

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	ResponseEntity<String> post(@RequestBody LocationData location) {
```
Comment (Gregam3) [inline backend/src/main/java/asegroup1/api/controllers/LocationController.java]: change `post()` to more descriptive name

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java (hunk 1 above)
Comment (Gregam3) [inline backend/src/main/java/asegroup1/api/controllers/LocationController.java]: For request mapping a value that corresponds to the chosen method name e.g. `/post` so the full endpoint would be `localhost:8080/location/post`

File: backend/src/main/java/asegroup1/api/daos/location/LocationDao.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package asegroup1.api.daos.location;
```
Comment (Gregam3) [inline backend/src/main/java/asegroup1/api/daos/location/LocationDao.java]: Can be deleted

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java (hunk 1 above)
Comment (jamesfernando94) [reply backend/src/main/java/asegroup1/api/controllers/LocationController.java]: changed

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java (hunk 2 above)
Comment (jamesfernando94) [reply backend/src/main/java/asegroup1/api/controllers/LocationController.java]: removed

File: backend/src/main/java/asegroup1/api/controllers/LocationController.java (hunk 3 above)
Comment (jamesfernando94) [reply backend/src/main/java/asegroup1/api/controllers/LocationController.java]: done changed to `postUpdateLocationData`

**Pull request merged**